README.md

```diff
@@ -179,6 +179,31 @@ Ensure you've followed the above development instructions and then:
 yarn test
 ```
 
+Running lint
```

Comment:
should these sections be in the contributing guide instead?

also we're mixing title styles: please let's not do that :p

Reply:
> also we're mixing title styles: please let's not do that :p

Fixed in https://github.com/matrix-org/matrix-react-sdk/pull/8705

Reply:
Apologies for the force push, I rebased on top of #8705